api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/constraint/EligibilityOverrideValidatorTest.java

```diff
@@ -47,14 +48,22 @@ void shouldFailToValidateEligibilityOverrideWithNullValues(EligibilityOverride e
                 EligibilityOverride.builder()
                         .eligibilityOutcome(null)
                         .overrideUntil(OVERRIDE_UNTIL_FIVE_YEARS)
+                        .childrenDob(emptyList())
```

Comment:
I think this would be clearer with specific dates like `MAGGIE_AND_LISA_DOBS` rather than `emptyList()`, same below too.